python/src/GridLayout_doc.i.in

```diff
@@ -144,3 +144,17 @@ nbColumns : int
 Notes
 -----
 If the new layout contains fewer graphs than the old layout, the remaining graphs are removed from the layout."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::GridLayout::add
+"Add subgraphs from a different grid to the corresponding subgraphs in the current grid.
+
+Parameters
+----------
+grid : :class:`~openturns.GridLayout`
+    GridLayout containing the subgraphs to add.
+
+Notes
+-----
+Only works if both grids have the same number of rows and columns."
```

Comment:
maybe you should be more explicit on what the method does. Seems difficult to catch the feature without reading the code

Reply:
https://openturns.github.io/openturns/latest/user_manual/_generated/openturns.Graph.html#openturns.Graph.add

Reply:
`:meth:`~openturns.SobolIndicesExperiment.generate`

Reply:
Done.